src/player.py

```diff
@@ -21,6 +29,15 @@ def add_examined_room(self, room_coordinate_x: int, room_coordinate_y: int) -> N
         else:
             self._examined_rooms.append(examined_room)
 
+    def move(self, next_room: Room) -> None:  # maybe return bool with alive status
+        """"""
+        if next_room.is_trap is True:
+            self._alive = False
+            return
+        self._coordinate_x = next_room.get_coordinate_x()
+        self._coordinate_y = next_room.get_coordinate_y()
```

Comment:
For such a small project it can be nice to have a shorter names for fields and properties, like:
`self._x = next_room.x()`